ui/src/app/utils/formatBytes.ts

```diff
@@ -1,27 +1,32 @@
 type Byte = { value: number; unit: string };
 
-type FormatBytesConfig = { binary?: boolean; precision?: number };
+type FormatBytesConfig = {
+  binary?: boolean;
+  convertTo?: string;
+  precision?: number;
+};
 
 /**
- * Format bytes to the appropriate unit at supplied precision.
+ * Format bytes to the appropriate/supplied unit at supplied precision.
  *
  * @param {number} value - the value in the supplied byte unit
  * @param {string} unit - the byte unit, e.g. "KB", "TB"
  * @param {object} config - config object
```

Comment:
`FormatBytesConfig` rather than `object` in jsdoc here

Reply:
Adding that now